tests/unit/test_error_handling.py

```diff
@@ -205,6 +206,45 @@ def test_handle_invalid_calculation_method_error():
         model.calculate_power_flow(calculation_method=CalculationMethod.iec60909)
 
 
+@pytest.mark.skip(reason="TODO: Automatic tap changer")
```

Comment:
maybe instead of `skip` use `xfail` so that we are notified whenever this actually passes

```suggestion
@pytest.mark.xfail(reason="TODO: Automatic tap changer")
```